server/src/main/protobuf/RegisteredArrivalMessage.proto

```diff
@@ -8,7 +8,8 @@ message RegisteredArrivalsMessage {
 
 message RegisteredArrivalMessage {
     optional string origin = 1;
-    optional string voyageNumber = 2;
-    optional int64 scheduled = 3;
-    optional int64 lookedUp = 4;
+    optional string destination = 2;
```

Comment:
Is this change part of the fixed points bug?

Reply:
nice rhetorical question :)